test/test_common/environment.h

```diff
@@ -93,6 +93,12 @@ class TestEnvironment {
    */
   static const std::string& nullDevicePath();
 
+  /**
+   * Obtain platform specific new line character(s)
+   * @return const std::string& platform specific new line character(s)
+   */
+  static const std::string& newLine();
```

Comment:
From its usage, this can be `constexpr absl::string_view` without constructing `std::string`.

Reply:
That's smart! thanks I will do it.

Reply:
fixed in bf596518c032a3a88657ef885bb7c0840eb348d2